lua/paq.lua

```diff
@@ -111,6 +113,17 @@ local function clone(pkg, counter, sync)
             counter(pkg.name, "err", sync)
         end
     end)
+    if pkg.commit then
+        call_proc("git", commit_args, nil, function(ok)
```

Comment:
I don't have much time today to work on this, but practically what I'm trying to do here is trigger a second git command basically using what you used to trigger the clone above.